openmc/lattice.py

```diff
@@ -869,11 +934,13 @@ def __init__(self, lattice_id=None, name=''):
         self._num_rings = None
         self._num_axial = None
         self._center = None
+        self._hextype = 0 # by Default orientaion is OY
```

Comment:
I find the use of 0 and 1 to be a bit confusing. I would just use a string and call this attribute `_orientation`:
```Python
self._orientation = 'y'
```

This will require other changes anywhere `self._hextype` is used.